packages/fast-components-react-msft/src/pivot/index.ts

```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+import MSFTPivot, {
+    PivotHandledProps as MSFTPivotHandledProps,
+    PivotManagedClasses,
+    PivotProps as MSFTPivotProps,
+    PivotUnhandledProps,
+} from "./pivot";
+import { PivotClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import manageJss, { ManagedJSSProps } from "@microsoft/fast-jss-manager-react";
+import { DesignSystem, PivotStyles } from "@microsoft/fast-components-styles-msft";
+import { Subtract } from "utility-types";
+
+/*tslint:disable-next-line:typedef */
+const Pivot = manageJss(PivotStyles)(MSFTPivot);
+type Pivot = InstanceType<typeof Pivot>;
+
+interface PivotHandledProps
+    extends Subtract<MSFTPivotHandledProps, PivotManagedClasses> {}
+type PivotProps = ManagedJSSProps<MSFTPivotProps, PivotClassNameContract, DesignSystem>;
+
+export * from "./pivot";

```

Comment:
I don't think you want this `export * from "./pivot"` statement.

With that statement, you are actually exporting two interfaces named `PivotHandledProps` with conflicting types. I think we want to be more intentional/explicit about what we're exporting from this index file.

Reply:
Got it, yeah I think all I need to do is moved PivotManagedClasses into the named export at the bottom of this file.